ts/src/deribit.ts

```diff
@@ -487,6 +487,9 @@ export default class deribit extends Exchange {
                 'fetchBalance': {
                     'code': 'BTC',
                 },
+                'fetchTickers': {
```

Comment:
why we need this?

Reply:
ah, was remnant from previous commit that i discarded.
removed those lines, now clean.